src/spectrecoff-cli/SpectreCoff.Cli.fsproj

```diff
@@ -15,6 +15,7 @@
     <Compile Include="commands/Rule.fs" />
     <Compile Include="commands/Figlet.fs" />
     <Compile Include="commands/BarChart.fs" />
+    <Compile Include="commands\Progress.fs" />
```

Comment:
Again wrong slash :P

Reply:
Not nitpick, totally justified :) this is my failure, thanks for pointing it out.